motion/ruby_motion_query/device.rb

```diff
@@ -92,7 +92,7 @@ def iphone?
       def simulator?
         if @_simulator.nil?
           @_simulator = if ios_at_least?(9.0)
-            !NSBundle.mainBundle.bundlePath.start_with?('/var/')
+            !(NSBundle.mainBundle.bundlePath =~ /Library\/Developer/i).nil?
```

Comment:
checking the history of the file we updated to this to support iOS9, so I wonder what changed that its now again changing still to support iOS 9 - maybe a sdk bump, or something?